### PR TITLE
Adjust ag-charts-enterprise entry point to inc. community

### PR DESCRIPTION
**Pull request by iMoses**
Mimic interfaces like `ag-grid`

### CODE REVIEW

File: packages/ag-charts-community/src/chart/test/load-example.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Logger } from '../../util/logger';
 
 export const filters = [
     /.*AgChart\.(update|create)/,
-    /.*AgEnterpriseCharts\.(update|create)/,
+    /.*AgChart\.(update|create)/,
```
Comment (iMoses) [inline packages/ag-charts-community/src/chart/test/load-example.ts]: i'll remove this duplication before merged

**Comment by iMoses** [comment]
BTW, do we wanna change the main object from `AgChart` to `AgCharts`?

**Comment by rmc-software** [comment]
@iMoses "BTW, do we wanna change the main object from AgChart to AgCharts?" - yes we should - breaking change obv. Perhaps we can also keep AgChart and deprecate it?

File: packages/ag-charts-community/project.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         "rollupConfig": "packages/ag-charts-community/rollup.config.cjs",
         "project": "packages/ag-charts-community/package.json",
         "compiler": "tsc",
-        "format": ["cjs"]
+        "format": ["cjs", "esm"]
```
Comment (alantreadway) [inline packages/ag-charts-community/project.json]: I'm a bit concenred about the performance impact here - is it doubling our reload times?

File: packages/ag-charts-enterprise/src/main.ts [hunk 3]
```diff
@@ -60,19 +60,11 @@ _ModuleSupport.registerModule(TreemapSeriesModule);
 _ModuleSupport.registerModule(WaterfallModule);
 _ModuleSupport.registerModule(ZoomModule);
 
-export class AgEnterpriseCharts {
-    public static create(options: AgChartOptions): AgChartInstance {
+export class AgChart extends AgChartCommunity {
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/main.ts]: Does this approach work for the FW cases, i.e. do we currently get the license message on opening an enterprise React example?

Comment (iMoses) [reply packages/ag-charts-enterprise/src/main.ts]: Haven't checked specific libraries, just plain Javascript. I'll verify tomorrow.

File: packages/ag-charts-community/project.json (hunk 2 above)
Comment (iMoses) [reply packages/ag-charts-community/project.json]: I don't think it's that bad. I'm also not sure if we have any other options.

**Pull request closed without merging**